service/src/main/scala/za/co/absa/logingw/rest/Application.scala

```diff
@@ -19,12 +19,31 @@ package za.co.absa.logingw.rest
 import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.databind.{ObjectMapper, SerializationFeature}
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType
+import io.swagger.v3.oas.annotations.info.{Info, License}
+import io.swagger.v3.oas.annotations.security.SecurityScheme
+import io.swagger.v3.oas.annotations.{ExternalDocumentation, OpenAPIDefinition}
 import org.springframework.boot.SpringApplication
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.builder.SpringApplicationBuilder
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer
 import org.springframework.context.annotation._
 
+@OpenAPIDefinition(
+  externalDocs = new ExternalDocumentation(description = "GitHub",
+    url = "https://github.com/AbsaOSS/login-gateway"
+  ),
+  info = new Info(
+    title = "Login Gateway API",
+    version = "0.1",
```

Comment:
Can these be filled from a config/sbt files? 
I presume they can't but I have to ask.

Reply:
I believe they can be, similar to what we do in other project (Enceladus does this somehow). To be part of #5.